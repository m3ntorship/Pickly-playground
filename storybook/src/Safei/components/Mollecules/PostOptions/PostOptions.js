import React from "react";
import Toggle from "../../Atoms/Toggles/Toggle";

const PostOptions = () => {
  return (
    <div className="flex flex-row w-full">
      <div className="mt-0 pb-2">
        <Toggle size="sm" />
      </div>
      <div
        className="text-xs ml-2 leading-4 mb-0 pr-6  border-r border-grey-shd6"
        style={{ paddingTop: "2px" }}
      >
        Hide My Identity
      </div>
    </div>
  );
};

export default PostOptions;
