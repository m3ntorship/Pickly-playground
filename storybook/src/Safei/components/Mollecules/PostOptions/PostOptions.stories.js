import React from "react";
import PostOptions from "./PostOptions";

export default {
  title: "Mollecules/PostOptions",
  component: PostOptions,
};

const Template = (args) => <PostOptions {...args} />;
export const _PostOptions = Template.bind({});
_PostOptions.args = {
  //   status: "error",
  // prefix: '+20',
  // direction: 'flex-row'
};
