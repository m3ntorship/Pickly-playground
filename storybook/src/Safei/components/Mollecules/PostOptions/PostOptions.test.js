import React from "react";
import renderer from "react-test-renderer";
import PostOptions from "./PostOptions";

describe("Test my PostOptions components", () => {
  //(1)
  it("renders a PostOptions bar ", () => {
    const tree = renderer.create(<PostOptions />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
