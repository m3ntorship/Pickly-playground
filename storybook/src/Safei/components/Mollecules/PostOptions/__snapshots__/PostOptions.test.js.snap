// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test my PostOptions components renders a PostOptions bar  1`] = `
<div
  className="flex flex-row w-full"
>
  <div
    className="mt-0 pb-2"
  >
    <div
      style={
        Object {
          "transform": "scale(0.666)",
        }
      }
    >
      <label
        className="switch"
      >
        <input
          type="checkbox"
        />
        <span
          className="slider round"
        />
      </label>
    </div>
  </div>
  <div
    className="text-xs ml-2 leading-4 mb-0 pr-6  border-r border-grey-shd6"
    style={
      Object {
        "paddingTop": "2px",
      }
    }
  >
    Hide My Identity
  </div>
  <div
    className="text-xs ml-2 leading-4 mb-0 pr-6 cursor-pointer"
    style={
      Object {
        "paddingTop": "2px",
      }
    }
  >
    <label>
      Privacy:
    </label>
    <select
      className="cursor-pointer"
      id="privacy"
      name="privacy"
    >
      <option
        value="Public"
      >
        Public
      </option>
      <option
        value="Custom"
      >
        Custom
      </option>
      <option
        value="Private"
      >
        Private
      </option>
    </select>
  </div>
</div>
`;
